### PR TITLE
Update resources page layout

### PR DESCRIPTION
**Pull request by fritzjooste**
The new layout includes more resources, gets rid of the checklist, and uses cards.

### CODE REVIEW

File: brigade/templates/organize/index.html [hunk 1]
```diff
@@ -3,19 +3,98 @@
...
-    <p><a href="checklist/">Brigade Organizer's Checklist</a>: Learn how Code for America supports civic hack nights and Official Brigades.</p>
-    <p><a href="playbook/">Brigade Organizer’s Playbook</a>: Deep dive into three areas key to running a sustainable Brigade.</p>
-    <p><a href="{{ url_for('.free_software_index') }}">Free Software Available to Brigades</a>: Learn about the software that vendors have donated for Brigades to use.</p>
 
-    <p><a href="https://cfa.typeform.com/to/uxPQH6" class="button button-l" target="_blank"  title="New-to-organizing form">New to organizing?</a> Tell us a little bit about yourself so we can help you get started.</p>
+    <div class="card-row">
+
+      <a href="https://docs.google.com/document/d/19bN5RWK5nQTpz0mHUViHrzHiommBUAMSztwNRzUcxYo/" class="card card--horizontal" target="_blank">
+        <div class="card__aside">
+          <div class="card__icon"><img src="{{ asset_url_for('images/icon_playbook.png') }}"></div>
```
Comment (tdooner) [inline brigade/templates/organize/index.html]: No font-awesome? :(

File: brigade/templates/organize/index.html [hunk 2]
```diff
@@ -3,19 +3,98 @@
...
+        <div class="card__body">
+          <h2 class="card__title">Brigade Contact Directory</h2>
+          <p>Want to reach out to a Brigade? Look them up in the up-to-date contact directory.</p>
+          <div class="card__action">
+            <div class="button">Use the Directory <i class="fas fa-external-link-alt"></i></div>
+          </div>
+        </div>
+      </a>
+      
+      <a href="https://docs.google.com/document/d/1c9Jbtdj_wKP2dDwxslXhBQoobzpTwsxkrSjAlLQdkI0/edit#heading=h.c8bc7axfomwl" class="card" target="_blank">
```
Comment (tdooner) [inline brigade/templates/organize/index.html]: nit: Add a title so we can track clicks to this link

File: brigade/templates/organize/index.html [hunk 3]
```diff
@@ -3,19 +3,98 @@
...
+        <div class="card__body">
+          <h2 class="card__title">Software Available to Brigades</h2>
+          <p>Learn about the software that vendors have donated for Brigades to use.</p>
+          <div class="card__action">
+            <div class="button">Explore Software</div>
+          </div>
+        </div>
+      </a>
+
+      <a href="http://c4a.me/brigades" class="card" target="_blank">
```
Comment (tdooner) [inline brigade/templates/organize/index.html]: nit: Add a title so we can track clicks to this link

File: brigade/templates/organize/index.html [hunk 4]
```diff
@@ -3,19 +3,98 @@
...
+          <h2 class="card__title">Organizer’s Playbook</h2>
+          <p>The Playbook gives practical advice and real examples on running a sustainable Brigade.</p>
+          <p><small>It includes: guidance on hosting events, building a core team, and practicing user-centered design.</small></p>
+          <div class="card__action">
+            <div class="button">View Playbook <i class="fas fa-external-link-alt"></i></div>
+          </div>
+        </div>
+      </a>
+
+      <a href="https://docs.google.com/document/d/16CL9TdmWV0hDY6c85PwtzUcu1VjeSeiDFD2CbtLKf7s/edit?usp=sharing" class="card card--horizontal" target="_blank">
```
Comment (tdooner) [inline brigade/templates/organize/index.html]: nitpick: Add a title so we can track clicks to this link

File: brigade/templates/organize/index.html (hunk 1 above)
Comment (fritzjooste) [reply brigade/templates/organize/index.html]: Couldn't find a Font Awesome icon that represented 'playbook' properly :(
One option is a clipboard: https://fontawesome.com/icons/clipboard?style=regular
What do you think? Is that better than the current image?

**Comment by tdooner** [comment]
🎉

**Pull request merged**